pod-master/podmaster.go

```diff
@@ -35,14 +35,15 @@ import (
 )
 
 type config struct {
-	etcdServers string
-	key         string
-	whoami      string
-	ttl         uint64
-	src         string
-	dest        string
-	sleep       time.Duration
-	lastLease   time.Time
+	etcdServers    string
+	etcdConfigFile string
```

Comment:
My feeling was that we are going to get rid of it in apiserver.

@timothysc @lavalamp - am I right?